packages/lib-subject-viewers/preview.js

```diff
@@ -0,0 +1,61 @@
+import { INITIAL_VIEWPORTS } from '@storybook/addon-viewport'
```

Comment:
Is this a duplicate of `/.storybook/preview.js`?

Reply:
Yup - good catch. Removed.